pages/dkp/kommander/2.2/dkp-upgrade/upgrade-kommander/index.md

```diff
@@ -5,43 +5,59 @@ title: Upgrade Kommander
 
 ## Prerequisites
 
-- [Download][download-binary] and install the latest DKP CLI binary on your computer.
+- [Download][download_binary] and install the latest DKP CLI binary on your computer.
 - Ensure you are on DKP version 2.1 or 2.1.1 and Kubernetes version 1.21.
-- If you have attached clusters, ensure they are on Kubernetes versions 1.19, 1.20 or 1.21. To upgrade your Kubernetes version, refer to the appropriate documentation for your environment: [AKS][AKS], [AWS][AWS], [Azure][Azure], [EKS][EKS], [pre-provisioned][pre-provisioned].
-- Review the [Platform Application version updates][platform-apps] that are part of this upgrade.
+- If you have attached clusters, ensure they are on Kubernetes versions 1.19, 1.20 or 1.21. To upgrade your Kubernetes version, refer to the appropriate documentation for your environment: [AKS][AKS], [AWS][AWS], [Azure][Azure], [EKS][EKS], [pre-provisioned][pre_provisioned].
+- Review the [Platform Application version updates][release_notes] that are part of this upgrade.
+- For air-gapped environments only: Download the [images bundle][images_bundle], [charts bundle][charts_bundle] and [Catalog Application charts bundle][cat_apps_bundle].  
+- For air-gapped environments only: [Load the Docker images into your Docker registry][load_images]
 
 ## Upgrade Kommander
 
-Before running the following command, ensure that your `dkp` configuration **references the Kommander Management cluster**, otherwise it attempts to run the upgrade on the bootstrap cluster. You can do this by setting the `KUBECONFIG` environment variable [to the appropriate kubeconfig file's location][k8s-access-to-clusters].
+Before running the following command, ensure that your `dkp` configuration **references the Kommander Management cluster**, otherwise it attempts to run the upgrade on the bootstrap cluster. You can do this by setting the `KUBECONFIG` environment variable [to the appropriate kubeconfig file's location][k8s_access_to_clusters].
 
-<p class="message--note"><strong>NOTE:</strong> An alternative to initializing the KUBECONFIG environment variable as stated earlier is to use the <code>–kubeconfig=cluster_name.conf</code> flag. This ensures that Kommander is upgraded on the workload cluster.</p>
+<p class="message--note"><strong>NOTE:</strong> An alternative to initializing the KUBECONFIG environment variable as stated earlier is to use the <code>–kubeconfig=cluster_name.conf</code> flag. This ensures that Kommander upgrades on the workload cluster.</p>
 
-1.  Use the DKP CLI to upgrade Kommander and all the Platform Applications in the Management cluster:
+1.  Use the DKP CLI to upgrade Kommander and all the Platform Applications in the Management Cluster:
+
+    For air-gapped:
+
+    ```bash
+    dkp upgrade kommander --charts-bundle bundle.tar.gz
```

Comment:
I think this will need to include the chart bundle and the updated kommander application repo 🤔. maybe @gracedo or @mikolajb can confirm?

```bash
dkp upgrade kommander --charts-bundle dkp-kommander-charts-bundle_${VERSION}.tar.gz --kommander-applications-repository kommander-applications_${VERSION}.tar.gz
```

Reply:
@takirala can you look this one over?

Reply:
So we have 3 different options here, since @makkes also proposed: 
`dkp upgrade kommander --charts-bundle bundle.tar.gz --kommander-applications-repository kommander-applications_v2.2.0.tar.gz`

Reply:
@gvasquezvargas both are correct! it just depends on what the bundles are named when they are downloaded. I'd suggest we stay consistent with what we document on this [page](https://dev-docs.d2iq.com/dkp/kommander/2.2/install/air-gapped/#load-the-docker-images-into-your-docker-registry) if possible.

On that page we have example commands that end with the bundles looking like:
```
--kommander-applications-repository kommander-applications_${VERSION}.tar.gz
--charts-bundle dkp-kommander-charts-bundle_${VERSION}.tar.gz
```

Reply:
@gvasquezvargas and I synced up about this.

If we use the commands from the [install page](https://dev-docs.d2iq.com/dkp/kommander/2.2/install/air-gapped/#install-on-konvoy) for downloading the bundles. Unfortunately, linking directly to the download links isn't working (and users may be more privy to downloading via the CLI anyway).

```
export VERSION=v2.2.0
```

Download Image bundle (we could link directly to this [page](https://dev-docs.d2iq.com/dkp/kommander/2.2/install/air-gapped/#load-the-docker-images-into-your-docker-registry))
```
wget "https://downloads.d2iq.com/dkp/${VERSION}/kommander_image_bundle_${VERSION}_linux_amd64.tar.gz" -O kommander-image-bundle.tar.gz
```

Download Kommander Application Definitions: (this doesn't have a direct link to an existing doc)
```
wget "https://downloads.d2iq.com/dkp/${VERSION}/kommander-applications_${VERSION}.tar.gz"
```

Kommander charts bundle: (this doesn't have a direct link to an existing doc)
```
wget "https://downloads.d2iq.com/dkp/${VERSION}/dkp-kommander-charts-bundle_${VERSION}.tar.gz"
```

Then users can copy/paste those and end up with the upgrade command being:

```
dkp upgrade kommander --charts-bundle dkp-kommander-charts-bundle_${VERSION}.tar.gz --kommander-applications-repository kommander-applications_${VERSION}.tar.gz
```

Reply:
what about 
- charts bundle for `dkp-catalog-applications`
- `dkp-insights-catalog`

i guess insights isn't ready yet but we can have a placeholder comment to add it in later?!